### ISSUE TITLE
2.0.0-alpha.1 - RangeError: Schema is missing its top node type ('doc')

### ISSUE THREAD

**Issue by arthabus** [Leecason/element-tiptap#464]
**Describe the bug**
Trying to setup an minimal version from the example and getting "RangeError: Schema is missing its top node type ('doc')"

**To Reproduce**
1. Follow the isntructions on the mai page for element-tiptap
2. Observe the issue

**Expected behavior**
It displays the editor with the inititla content

**Screenshots**

**Desktop (please complete the following information):**
 - OS: Max Os 13.2 (22D49)
 - Browser: Chrome 113.0.5672.126 (Official Build) (x86_64)
 - Version 2.0.0-alpha.1

**Additional context**

I'm using webpack

**Comment by okijhhyu** [comment]
**Document**!!! not doc

**Comment by arthabus** [comment]
Yes, it's Document. And the readme for alpha needs to be fixed, as in the readme it says "Doc" under the "usage" section

**Comment by vesper8** [comment]
This version fixes all the issues: https://github.com/okijhhyu/element-tiptap-vue3 :)
